CRITICAL - 2024-04-23 15:50:12 --> ParseError: syntax error, unexpected end of file
[Method: GET, Route: dmdrdv]
in APPPATH\Config\Routes.php on line 30.
 1 SYSTEMPATH\CodeIgniter.php(823): CodeIgniter\Router\RouteCollection->loadRoutes()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-23 16:00:57 --> ErrorException: Undefined variable $users
[Method: GET, Route: dmdrdv]
in APPPATH\Views\Dmdrdv.php on line 46.
 1 APPPATH\Views\Dmdrdv.php(46): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $users', 'C:\\xampp\\htdocs\\Gestion\\app\\Views\\Dmdrdv.php', 46)
 2 SYSTEMPATH\View\View.php(234): include('C:\\xampp\\htdocs\\Gestion\\app\\Views\\Dmdrdv.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('Dmdrdv', [], true)
 5 APPPATH\Controllers\dmdrdvController.php(17): view('Dmdrdv')
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\dmdrdvController->dmdrdv()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\dmdrdvController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-23 16:01:56 --> ErrorException: Undefined variable $users
[Method: GET, Route: dmdrdv]
in APPPATH\Views\Dmdrdv.php on line 46.
 1 APPPATH\Views\Dmdrdv.php(46): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $users', 'C:\\xampp\\htdocs\\Gestion\\app\\Views\\Dmdrdv.php', 46)
 2 SYSTEMPATH\View\View.php(234): include('C:\\xampp\\htdocs\\Gestion\\app\\Views\\Dmdrdv.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('Dmdrdv', [], true)
 5 APPPATH\Controllers\dmdrdvController.php(17): view('Dmdrdv')
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\dmdrdvController->dmdrdv()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\dmdrdvController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-23 16:01:58 --> ErrorException: Undefined variable $users
[Method: GET, Route: dmdrdv]
in APPPATH\Views\Dmdrdv.php on line 46.
 1 APPPATH\Views\Dmdrdv.php(46): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $users', 'C:\\xampp\\htdocs\\Gestion\\app\\Views\\Dmdrdv.php', 46)
 2 SYSTEMPATH\View\View.php(234): include('C:\\xampp\\htdocs\\Gestion\\app\\Views\\Dmdrdv.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('Dmdrdv', [], true)
 5 APPPATH\Controllers\dmdrdvController.php(17): view('Dmdrdv')
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\dmdrdvController->dmdrdv()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\dmdrdvController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-23 16:03:11 --> ErrorException: Undefined array key "nom"
[Method: GET, Route: dmdrdv]
in APPPATH\Views\Dmdrdv.php on line 47.
 1 APPPATH\Views\Dmdrdv.php(47): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "nom"', 'C:\\xampp\\htdocs\\Gestion\\app\\Views\\Dmdrdv.php', 47)
 2 SYSTEMPATH\View\View.php(234): include('C:\\xampp\\htdocs\\Gestion\\app\\Views\\Dmdrdv.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('Dmdrdv', [], true)
 5 APPPATH\Controllers\dmdrdvController.php(17): view('Dmdrdv', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\dmdrdvController->dmdrdv()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\dmdrdvController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-23 16:06:14 --> ErrorException: Undefined array key "nom"
[Method: GET, Route: dmdrdv]
in APPPATH\Views\Dmdrdv.php on line 49.
 1 APPPATH\Views\Dmdrdv.php(49): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "nom"', 'C:\\xampp\\htdocs\\Gestion\\app\\Views\\Dmdrdv.php', 49)
 2 SYSTEMPATH\View\View.php(234): include('C:\\xampp\\htdocs\\Gestion\\app\\Views\\Dmdrdv.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('Dmdrdv', [], true)
 5 APPPATH\Controllers\dmdrdvController.php(17): view('Dmdrdv', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\dmdrdvController->dmdrdv()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\dmdrdvController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-04-23 16:19:41 --> mysqli_sql_exception: Unknown column 'id_user' in 'field list' in C:\xampp\htdocs\Gestion\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\Gestion\system\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `dm...', 0)
#1 C:\xampp\htdocs\Gestion\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `dm...')
#2 C:\xampp\htdocs\Gestion\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `dm...')
#3 C:\xampp\htdocs\Gestion\system\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `dm...', Array, false)
#4 C:\xampp\htdocs\Gestion\system\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\Gestion\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\Gestion\system\Model.php(791): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\Gestion\app\Controllers\dmdrdvController.php(32): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\Gestion\system\CodeIgniter.php(933): App\Controllers\dmdrdvController->processDemande()
#9 C:\xampp\htdocs\Gestion\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\dmdrdvController))
#10 C:\xampp\htdocs\Gestion\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\Gestion\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\Gestion\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\Gestion\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-04-23 16:19:41 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'id_user' in 'field list'
[Method: POST, Route: dmdrdv/processDemande]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `dmdrdv` (`sujet`, `id_user`, `date_demande`) VALUES (:sujet:, :id_user:, :date_demande:)', [...], false)
 2 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\dmdrdvController.php(32): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\dmdrdvController->processDemande()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\dmdrdvController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-23 16:19:41 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'id_user' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `dmdrdv` (`sujet`, `id_user`, `date_demande`) VALUES (\'douleur a la jambe\', \'39\', \'2024-04-23 16:19:41\')')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `dmdrdv` (`sujet`, `id_user`, `date_demande`) VALUES (\'douleur a la jambe\', \'39\', \'2024-04-23 16:19:41\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `dmdrdv` (`sujet`, `id_user`, `date_demande`) VALUES (:sujet:, :id_user:, :date_demande:)', [...], false)
 4 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\dmdrdvController.php(32): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\dmdrdvController->processDemande()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\dmdrdvController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-23 16:19:41 --> [Caused by] mysqli_sql_exception: Unknown column 'id_user' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `dmdrdv` (`sujet`, `id_user`, `date_demande`) VALUES (\'douleur a la jambe\', \'39\', \'2024-04-23 16:19:41\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `dmdrdv` (`sujet`, `id_user`, `date_demande`) VALUES (\'douleur a la jambe\', \'39\', \'2024-04-23 16:19:41\')')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `dmdrdv` (`sujet`, `id_user`, `date_demande`) VALUES (\'douleur a la jambe\', \'39\', \'2024-04-23 16:19:41\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `dmdrdv` (`sujet`, `id_user`, `date_demande`) VALUES (:sujet:, :id_user:, :date_demande:)', [...], false)
 5 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\dmdrdvController.php(32): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\dmdrdvController->processDemande()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\dmdrdvController))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-04-23 16:23:16 --> mysqli_sql_exception: Unknown column 'date_demande' in 'field list' in C:\xampp\htdocs\Gestion\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\Gestion\system\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `dm...', 0)
#1 C:\xampp\htdocs\Gestion\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `dm...')
#2 C:\xampp\htdocs\Gestion\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `dm...')
#3 C:\xampp\htdocs\Gestion\system\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `dm...', Array, false)
#4 C:\xampp\htdocs\Gestion\system\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\Gestion\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\Gestion\system\Model.php(791): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\Gestion\app\Controllers\dmdrdvController.php(32): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\Gestion\system\CodeIgniter.php(933): App\Controllers\dmdrdvController->processDemande()
#9 C:\xampp\htdocs\Gestion\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\dmdrdvController))
#10 C:\xampp\htdocs\Gestion\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\Gestion\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\Gestion\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\Gestion\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-04-23 16:23:16 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'date_demande' in 'field list'
[Method: POST, Route: dmdrdv/processDemande]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `dmdrdv` (`sujet`, `date_demande`) VALUES (:sujet:, :date_demande:)', [...], false)
 2 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\dmdrdvController.php(32): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\dmdrdvController->processDemande()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\dmdrdvController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-23 16:23:16 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'date_demande' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `dmdrdv` (`sujet`, `date_demande`) VALUES (\'douleur a la jambe\', \'2024-04-23 16:23:16\')')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `dmdrdv` (`sujet`, `date_demande`) VALUES (\'douleur a la jambe\', \'2024-04-23 16:23:16\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `dmdrdv` (`sujet`, `date_demande`) VALUES (:sujet:, :date_demande:)', [...], false)
 4 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\dmdrdvController.php(32): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\dmdrdvController->processDemande()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\dmdrdvController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-23 16:23:16 --> [Caused by] mysqli_sql_exception: Unknown column 'date_demande' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `dmdrdv` (`sujet`, `date_demande`) VALUES (\'douleur a la jambe\', \'2024-04-23 16:23:16\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `dmdrdv` (`sujet`, `date_demande`) VALUES (\'douleur a la jambe\', \'2024-04-23 16:23:16\')')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `dmdrdv` (`sujet`, `date_demande`) VALUES (\'douleur a la jambe\', \'2024-04-23 16:23:16\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `dmdrdv` (`sujet`, `date_demande`) VALUES (:sujet:, :date_demande:)', [...], false)
 5 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\dmdrdvController.php(32): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\dmdrdvController->processDemande()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\dmdrdvController))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-04-23 16:27:01 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`hopital`.`dmdrdv`, CONSTRAINT `fk_user_id` FOREIGN KEY (`user`) REFERENCES `users` (`id`)) in C:\xampp\htdocs\Gestion\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\Gestion\system\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `dm...', 0)
#1 C:\xampp\htdocs\Gestion\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `dm...')
#2 C:\xampp\htdocs\Gestion\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `dm...')
#3 C:\xampp\htdocs\Gestion\system\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `dm...', Array, false)
#4 C:\xampp\htdocs\Gestion\system\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\Gestion\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\Gestion\system\Model.php(791): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\Gestion\app\Controllers\dmdrdvController.php(32): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\Gestion\system\CodeIgniter.php(933): App\Controllers\dmdrdvController->processDemande()
#9 C:\xampp\htdocs\Gestion\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\dmdrdvController))
#10 C:\xampp\htdocs\Gestion\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\Gestion\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\Gestion\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\Gestion\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-04-23 16:27:01 --> CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`hopital`.`dmdrdv`, CONSTRAINT `fk_user_id` FOREIGN KEY (`user`) REFERENCES `users` (`id`))
[Method: POST, Route: dmdrdv/processDemande]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `dmdrdv` (`sujet`, `date_demande`) VALUES (:sujet:, :date_demande:)', [...], false)
 2 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\dmdrdvController.php(32): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\dmdrdvController->processDemande()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\dmdrdvController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-23 16:27:01 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`hopital`.`dmdrdv`, CONSTRAINT `fk_user_id` FOREIGN KEY (`user`) REFERENCES `users` (`id`))
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `dmdrdv` (`sujet`, `date_demande`) VALUES (\'douleur a la jambe\', \'2024-04-23 16:27:01\')')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `dmdrdv` (`sujet`, `date_demande`) VALUES (\'douleur a la jambe\', \'2024-04-23 16:27:01\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `dmdrdv` (`sujet`, `date_demande`) VALUES (:sujet:, :date_demande:)', [...], false)
 4 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\dmdrdvController.php(32): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\dmdrdvController->processDemande()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\dmdrdvController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-23 16:27:01 --> [Caused by] mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`hopital`.`dmdrdv`, CONSTRAINT `fk_user_id` FOREIGN KEY (`user`) REFERENCES `users` (`id`))
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `dmdrdv` (`sujet`, `date_demande`) VALUES (\'douleur a la jambe\', \'2024-04-23 16:27:01\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `dmdrdv` (`sujet`, `date_demande`) VALUES (\'douleur a la jambe\', \'2024-04-23 16:27:01\')')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `dmdrdv` (`sujet`, `date_demande`) VALUES (\'douleur a la jambe\', \'2024-04-23 16:27:01\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `dmdrdv` (`sujet`, `date_demande`) VALUES (:sujet:, :date_demande:)', [...], false)
 5 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\dmdrdvController.php(32): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\dmdrdvController->processDemande()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\dmdrdvController))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-04-23 16:27:23 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`hopital`.`dmdrdv`, CONSTRAINT `fk_user_id` FOREIGN KEY (`user`) REFERENCES `users` (`id`)) in C:\xampp\htdocs\Gestion\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\Gestion\system\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `dm...', 0)
#1 C:\xampp\htdocs\Gestion\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `dm...')
#2 C:\xampp\htdocs\Gestion\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `dm...')
#3 C:\xampp\htdocs\Gestion\system\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `dm...', Array, false)
#4 C:\xampp\htdocs\Gestion\system\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\Gestion\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\Gestion\system\Model.php(791): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\Gestion\app\Controllers\dmdrdvController.php(32): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\Gestion\system\CodeIgniter.php(933): App\Controllers\dmdrdvController->processDemande()
#9 C:\xampp\htdocs\Gestion\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\dmdrdvController))
#10 C:\xampp\htdocs\Gestion\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\Gestion\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\Gestion\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\Gestion\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-04-23 16:27:23 --> CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`hopital`.`dmdrdv`, CONSTRAINT `fk_user_id` FOREIGN KEY (`user`) REFERENCES `users` (`id`))
[Method: POST, Route: dmdrdv/processDemande]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `dmdrdv` (`sujet`, `date_demande`) VALUES (:sujet:, :date_demande:)', [...], false)
 2 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\dmdrdvController.php(32): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\dmdrdvController->processDemande()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\dmdrdvController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-23 16:27:23 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`hopital`.`dmdrdv`, CONSTRAINT `fk_user_id` FOREIGN KEY (`user`) REFERENCES `users` (`id`))
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `dmdrdv` (`sujet`, `date_demande`) VALUES (\'douleur a la jambe\', \'2024-04-23 16:27:23\')')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `dmdrdv` (`sujet`, `date_demande`) VALUES (\'douleur a la jambe\', \'2024-04-23 16:27:23\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `dmdrdv` (`sujet`, `date_demande`) VALUES (:sujet:, :date_demande:)', [...], false)
 4 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\dmdrdvController.php(32): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\dmdrdvController->processDemande()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\dmdrdvController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-23 16:27:23 --> [Caused by] mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`hopital`.`dmdrdv`, CONSTRAINT `fk_user_id` FOREIGN KEY (`user`) REFERENCES `users` (`id`))
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `dmdrdv` (`sujet`, `date_demande`) VALUES (\'douleur a la jambe\', \'2024-04-23 16:27:23\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `dmdrdv` (`sujet`, `date_demande`) VALUES (\'douleur a la jambe\', \'2024-04-23 16:27:23\')')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `dmdrdv` (`sujet`, `date_demande`) VALUES (\'douleur a la jambe\', \'2024-04-23 16:27:23\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `dmdrdv` (`sujet`, `date_demande`) VALUES (:sujet:, :date_demande:)', [...], false)
 5 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\dmdrdvController.php(32): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\dmdrdvController->processDemande()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\dmdrdvController))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-04-23 16:27:26 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`hopital`.`dmdrdv`, CONSTRAINT `fk_user_id` FOREIGN KEY (`user`) REFERENCES `users` (`id`)) in C:\xampp\htdocs\Gestion\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\Gestion\system\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `dm...', 0)
#1 C:\xampp\htdocs\Gestion\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `dm...')
#2 C:\xampp\htdocs\Gestion\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `dm...')
#3 C:\xampp\htdocs\Gestion\system\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `dm...', Array, false)
#4 C:\xampp\htdocs\Gestion\system\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\Gestion\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\Gestion\system\Model.php(791): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\Gestion\app\Controllers\dmdrdvController.php(32): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\Gestion\system\CodeIgniter.php(933): App\Controllers\dmdrdvController->processDemande()
#9 C:\xampp\htdocs\Gestion\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\dmdrdvController))
#10 C:\xampp\htdocs\Gestion\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\Gestion\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\Gestion\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\Gestion\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-04-23 16:27:26 --> CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`hopital`.`dmdrdv`, CONSTRAINT `fk_user_id` FOREIGN KEY (`user`) REFERENCES `users` (`id`))
[Method: POST, Route: dmdrdv/processDemande]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `dmdrdv` (`sujet`, `date_demande`) VALUES (:sujet:, :date_demande:)', [...], false)
 2 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\dmdrdvController.php(32): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\dmdrdvController->processDemande()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\dmdrdvController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-23 16:27:26 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`hopital`.`dmdrdv`, CONSTRAINT `fk_user_id` FOREIGN KEY (`user`) REFERENCES `users` (`id`))
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `dmdrdv` (`sujet`, `date_demande`) VALUES (\'douleur a la jambe\', \'2024-04-23 16:27:26\')')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `dmdrdv` (`sujet`, `date_demande`) VALUES (\'douleur a la jambe\', \'2024-04-23 16:27:26\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `dmdrdv` (`sujet`, `date_demande`) VALUES (:sujet:, :date_demande:)', [...], false)
 4 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\dmdrdvController.php(32): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\dmdrdvController->processDemande()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\dmdrdvController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-23 16:27:26 --> [Caused by] mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`hopital`.`dmdrdv`, CONSTRAINT `fk_user_id` FOREIGN KEY (`user`) REFERENCES `users` (`id`))
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `dmdrdv` (`sujet`, `date_demande`) VALUES (\'douleur a la jambe\', \'2024-04-23 16:27:26\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `dmdrdv` (`sujet`, `date_demande`) VALUES (\'douleur a la jambe\', \'2024-04-23 16:27:26\')')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `dmdrdv` (`sujet`, `date_demande`) VALUES (\'douleur a la jambe\', \'2024-04-23 16:27:26\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `dmdrdv` (`sujet`, `date_demande`) VALUES (:sujet:, :date_demande:)', [...], false)
 5 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\dmdrdvController.php(32): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\dmdrdvController->processDemande()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\dmdrdvController))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-23 17:08:38 --> ErrorException: Undefined variable $rdvs
[Method: GET, Route: rdv]
in APPPATH\Views\Rdv.php on line 51.
 1 APPPATH\Views\Rdv.php(51): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $rdvs', 'C:\\xampp\\htdocs\\Gestion\\app\\Views\\Rdv.php', 51)
 2 SYSTEMPATH\View\View.php(234): include('C:\\xampp\\htdocs\\Gestion\\app\\Views\\Rdv.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('Rdv', [], true)
 5 APPPATH\Controllers\rdvController.php(11): view('Rdv')
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\rdvController->rdv()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\rdvController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-23 17:09:43 --> ErrorException: Undefined variable $rdvs
[Method: GET, Route: rdv]
in APPPATH\Views\Rdv.php on line 51.
 1 APPPATH\Views\Rdv.php(51): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $rdvs', 'C:\\xampp\\htdocs\\Gestion\\app\\Views\\Rdv.php', 51)
 2 SYSTEMPATH\View\View.php(234): include('C:\\xampp\\htdocs\\Gestion\\app\\Views\\Rdv.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('Rdv', [], true)
 5 APPPATH\Controllers\rdvController.php(11): view('Rdv')
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\rdvController->rdv()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\rdvController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-23 17:11:46 --> ErrorException: Undefined variable $rdvs
[Method: GET, Route: rdv]
in APPPATH\Views\Rdv.php on line 51.
 1 APPPATH\Views\Rdv.php(51): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $rdvs', 'C:\\xampp\\htdocs\\Gestion\\app\\Views\\Rdv.php', 51)
 2 SYSTEMPATH\View\View.php(234): include('C:\\xampp\\htdocs\\Gestion\\app\\Views\\Rdv.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('Rdv', [], true)
 5 APPPATH\Controllers\rdvController.php(18): view('Rdv')
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\rdvController->rdv()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\rdvController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-23 17:31:04 --> ErrorException: Trying to access array offset on value of type null
[Method: GET, Route: rdv]
in APPPATH\Controllers\RdvController.php on line 12.
 1 APPPATH\Controllers\RdvController.php(12): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp\\htdocs\\Gestion\\app\\Controllers\\RdvController.php', 12)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RdvController->rdv()
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RdvController))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-23 17:31:08 --> ErrorException: Trying to access array offset on value of type null
[Method: GET, Route: rdv]
in APPPATH\Controllers\RdvController.php on line 12.
 1 APPPATH\Controllers\RdvController.php(12): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp\\htdocs\\Gestion\\app\\Controllers\\RdvController.php', 12)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RdvController->rdv()
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RdvController))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-23 17:31:09 --> ErrorException: Trying to access array offset on value of type null
[Method: GET, Route: rdv]
in APPPATH\Controllers\RdvController.php on line 12.
 1 APPPATH\Controllers\RdvController.php(12): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp\\htdocs\\Gestion\\app\\Controllers\\RdvController.php', 12)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RdvController->rdv()
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RdvController))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-23 17:31:11 --> ErrorException: Trying to access array offset on value of type null
[Method: GET, Route: rdv]
in APPPATH\Controllers\RdvController.php on line 12.
 1 APPPATH\Controllers\RdvController.php(12): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp\\htdocs\\Gestion\\app\\Controllers\\RdvController.php', 12)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RdvController->rdv()
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RdvController))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-23 17:31:15 --> ErrorException: Trying to access array offset on value of type null
[Method: GET, Route: rdv]
in APPPATH\Controllers\RdvController.php on line 12.
 1 APPPATH\Controllers\RdvController.php(12): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp\\htdocs\\Gestion\\app\\Controllers\\RdvController.php', 12)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RdvController->rdv()
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RdvController))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-23 18:12:30 --> ErrorException: Trying to access array offset on value of type null
[Method: GET, Route: confirmation]
in APPPATH\Controllers\ConfirmationController.php on line 13.
 1 APPPATH\Controllers\ConfirmationController.php(13): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp\\htdocs\\Gestion\\app\\Controllers\\ConfirmationController.php', 13)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ConfirmationController->conf()
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ConfirmationController))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-04-23 18:14:50 --> Email: Unable to send email using SMTP. Your server might not be configured to send mail using this method.
ERROR - 2024-04-23 18:14:50 --> 220 smtp.gmail.com ESMTP qt3-20020a170906ece300b00a5887fed95dsm324543ejb.2 - gsmtp

hello: 250-smtp.gmail.com at your service, [109.19.153.222]
250-SIZE 35882577
250-8BITMIME
250-STARTTLS
250-ENHANCEDSTATUSCODES
250-PIPELINING
250-CHUNKING
250 SMTPUTF8

starttls: 220 2.0.0 Ready to start TLS

hello: 250-smtp.gmail.com at your service, [109.19.153.222]
250-SIZE 35882577
250-8BITMIME
250-AUTH LOGIN PLAIN XOAUTH2 PLAIN-CLIENTTOKEN OAUTHBEARER XOAUTH
250-ENHANCEDSTATUSCODES
250-PIPELINING
250-CHUNKING
250 SMTPUTF8

Failed to authenticate password. Error: 535-5.7.8 Username and Password not accepted. For more information, go to
535 5.7.8  https://support.google.com/mail/?p=BadCredentials qt3-20020a170906ece300b00a5887fed95dsm324543ejb.2 - gsmtp

ERROR - 2024-04-23 18:24:31 --> Email: Unable to send email using SMTP. Your server might not be configured to send mail using this method.
ERROR - 2024-04-23 18:24:31 --> 220 smtp.gmail.com ESMTP h25-20020a1709070b1900b00a558f01499asm5328425ejl.154 - gsmtp

hello: 250-smtp.gmail.com at your service, [109.19.153.222]
250-SIZE 35882577
250-8BITMIME
250-STARTTLS
250-ENHANCEDSTATUSCODES
250-PIPELINING
250-CHUNKING
250 SMTPUTF8

starttls: 220 2.0.0 Ready to start TLS

hello: 250-smtp.gmail.com at your service, [109.19.153.222]
250-SIZE 35882577
250-8BITMIME
250-AUTH LOGIN PLAIN XOAUTH2 PLAIN-CLIENTTOKEN OAUTHBEARER XOAUTH
250-ENHANCEDSTATUSCODES
250-PIPELINING
250-CHUNKING
250 SMTPUTF8

Failed to authenticate password. Error: 535-5.7.8 Username and Password not accepted. For more information, go to
535 5.7.8  https://support.google.com/mail/?p=BadCredentials h25-20020a1709070b1900b00a558f01499asm5328425ejl.154 - gsmtp

CRITICAL - 2024-04-23 18:28:44 --> ErrorException: mail(): Failed to connect to mailserver at &quot;localhost&quot; port 25, verify your &quot;SMTP&quot; and &quot;smtp_port&quot; setting in php.ini or use ini_set()
[Method: POST, Route: dmdrdv/processDemande]
in APPPATH\Controllers\dmdrdvController.php on line 54.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mail(): Failed to connect to mailserver at &quot;localhost&quot; port 25, verify your &quot;SMTP&quot; and &quot;smtp_port&quot; setting in php.ini or use ini_set()', 'C:\\xampp\\htdocs\\Gestion\\app\\Controllers\\dmdrdvController.php', 54)
 2 APPPATH\Controllers\dmdrdvController.php(54): mail('ianaouchiche6@gmail.com', 'Confirmation de rendez-vous', 'Votre rendez-vous a été confirmé avec succès. Merci de votre confiance.', 'From: votre-adresse@example.com
Reply-To: votre-adresse@example.com
X-Mailer: PHP/8.2.12')
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\dmdrdvController->processDemande()
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\dmdrdvController))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-23 20:19:54 --> ParseError: syntax error, unexpected end of file, expecting "elseif" or "else" or "endif"
[Method: GET, Route: dashboard]
in APPPATH\Views\DashBoard.php on line 62.
 1 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('Dashboard', [], true)
 3 APPPATH\Controllers\DashBoardController.php(9): view('Dashboard')
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\DashboardController->dash()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DashboardController))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-23 21:05:32 --> ErrorException: Undefined variable $users
[Method: GET, Route: gestion]
in APPPATH\Views\GestionAdmin.php on line 38.
 1 APPPATH\Views\GestionAdmin.php(38): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $users', 'C:\\xampp\\htdocs\\Gestion\\app\\Views\\GestionAdmin.php', 38)
 2 SYSTEMPATH\View\View.php(234): include('C:\\xampp\\htdocs\\Gestion\\app\\Views\\GestionAdmin.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('GestionAdmin', [], true)
 5 APPPATH\Controllers\GestionController.php(9): view('GestionAdmin')
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\GestionController->gest()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\GestionController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-23 21:08:11 --> BadMethodCallException: Call to undefined method App\Models\User_model::getAllUsers
[Method: GET, Route: gestion]
in SYSTEMPATH\Model.php on line 921.
 1 APPPATH\Controllers\GestionController.php(11): CodeIgniter\Model->__call('getAllUsers', [])
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\GestionController->gest()
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\GestionController))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-23 21:09:12 --> ErrorException: Undefined array key "nom"
[Method: GET, Route: gestion]
in APPPATH\Views\GestionAdmin.php on line 39.
 1 APPPATH\Views\GestionAdmin.php(39): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "nom"', 'C:\\xampp\\htdocs\\Gestion\\app\\Views\\GestionAdmin.php', 39)
 2 SYSTEMPATH\View\View.php(234): include('C:\\xampp\\htdocs\\Gestion\\app\\Views\\GestionAdmin.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('GestionAdmin', [], true)
 5 APPPATH\Controllers\GestionController.php(12): view('GestionAdmin', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\GestionController->gest()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\GestionController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-23 21:10:53 --> ErrorException: Undefined variable $users
[Method: GET, Route: gestion]
in APPPATH\Views\GestionAdmin.php on line 38.
 1 APPPATH\Views\GestionAdmin.php(38): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $users', 'C:\\xampp\\htdocs\\Gestion\\app\\Views\\GestionAdmin.php', 38)
 2 SYSTEMPATH\View\View.php(234): include('C:\\xampp\\htdocs\\Gestion\\app\\Views\\GestionAdmin.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('GestionAdmin', [], true)
 5 APPPATH\Controllers\GestionController.php(13): view('GestionAdmin')
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\GestionController->gest()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\GestionController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-23 21:10:58 --> ErrorException: Undefined variable $users
[Method: GET, Route: gestion]
in APPPATH\Views\GestionAdmin.php on line 38.
 1 APPPATH\Views\GestionAdmin.php(38): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $users', 'C:\\xampp\\htdocs\\Gestion\\app\\Views\\GestionAdmin.php', 38)
 2 SYSTEMPATH\View\View.php(234): include('C:\\xampp\\htdocs\\Gestion\\app\\Views\\GestionAdmin.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('GestionAdmin', [], true)
 5 APPPATH\Controllers\GestionController.php(12): view('GestionAdmin')
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\GestionController->gest()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\GestionController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-23 21:10:59 --> ErrorException: Undefined variable $users
[Method: GET, Route: gestion]
in APPPATH\Views\GestionAdmin.php on line 38.
 1 APPPATH\Views\GestionAdmin.php(38): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $users', 'C:\\xampp\\htdocs\\Gestion\\app\\Views\\GestionAdmin.php', 38)
 2 SYSTEMPATH\View\View.php(234): include('C:\\xampp\\htdocs\\Gestion\\app\\Views\\GestionAdmin.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('GestionAdmin', [], true)
 5 APPPATH\Controllers\GestionController.php(12): view('GestionAdmin')
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\GestionController->gest()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\GestionController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-23 21:14:26 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "GestionAdmin/gest.php"
[Method: GET, Route: gestion]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 41.
 1 SYSTEMPATH\View\View.php(222): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('GestionAdmin/gest.php')
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('GestionAdmin/gest', [], true)
 3 APPPATH\Controllers\GestionController.php(18): view('GestionAdmin/gest', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\GestionController->gest()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\GestionController))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-23 21:28:00 --> ParseError: syntax error, unexpected single-quoted string "emploi", expecting "]"
[Method: GET, Route: gestion]
in APPPATH\Controllers\GestionController.php on line 37.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\Gestion\\app\\Controllers\\GestionController.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\GestionController')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\GestionController')
 4 SYSTEMPATH\CodeIgniter.php(887): class_exists('\\App\\Controllers\\GestionController', true)
 5 SYSTEMPATH\CodeIgniter.php(496): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-23 22:00:33 --> CodeIgniter\Database\Exceptions\DatabaseException: Updates are not allowed unless they contain a "where" or "like" clause.
[Method: POST, Route: gestion/modifier]
in SYSTEMPATH\Model.php on line 445.
 1 SYSTEMPATH\BaseModel.php(1028): CodeIgniter\Model->doUpdate(null, [...])
 2 SYSTEMPATH\Model.php(854): CodeIgniter\BaseModel->update(null, [...])
 3 APPPATH\Controllers\GestionController.php(100): CodeIgniter\Model->update(null, [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\GestionController->modifier()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\GestionController))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-23 22:00:51 --> CodeIgniter\Database\Exceptions\DatabaseException: Updates are not allowed unless they contain a "where" or "like" clause.
[Method: POST, Route: gestion/modifier]
in SYSTEMPATH\Model.php on line 445.
 1 SYSTEMPATH\BaseModel.php(1028): CodeIgniter\Model->doUpdate(null, [...])
 2 SYSTEMPATH\Model.php(854): CodeIgniter\BaseModel->update(null, [...])
 3 APPPATH\Controllers\GestionController.php(100): CodeIgniter\Model->update(null, [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\GestionController->modifier()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\GestionController))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-23 22:05:57 --> CodeIgniter\Database\Exceptions\DatabaseException: Updates are not allowed unless they contain a "where" or "like" clause.
[Method: POST, Route: gestion/modifier]
in SYSTEMPATH\Model.php on line 445.
 1 SYSTEMPATH\BaseModel.php(1028): CodeIgniter\Model->doUpdate(null, [...])
 2 SYSTEMPATH\Model.php(854): CodeIgniter\BaseModel->update(null, [...])
 3 APPPATH\Controllers\GestionController.php(102): CodeIgniter\Model->update(null, [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\GestionController->modifier()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\GestionController))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-23 22:08:09 --> CodeIgniter\Database\Exceptions\DatabaseException: Updates are not allowed unless they contain a "where" or "like" clause.
[Method: POST, Route: gestion/modifier]
in SYSTEMPATH\Model.php on line 445.
 1 SYSTEMPATH\BaseModel.php(1028): CodeIgniter\Model->doUpdate(null, [...])
 2 SYSTEMPATH\Model.php(854): CodeIgniter\BaseModel->update(null, [...])
 3 APPPATH\Controllers\GestionController.php(105): CodeIgniter\Model->update(null, [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\GestionController->modifier()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\GestionController))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-23 22:18:13 --> ErrorException: Undefined variable $user
[Method: GET, Route: modifier]
in APPPATH\Views\ModifierUtilisateur.php on line 4.
 1 APPPATH\Views\ModifierUtilisateur.php(4): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $user', 'C:\\xampp\\htdocs\\Gestion\\app\\Views\\ModifierUtilisateur.php', 4)
 2 SYSTEMPATH\View\View.php(234): include('C:\\xampp\\htdocs\\Gestion\\app\\Views\\ModifierUtilisateur.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('ModifierUtilisateur', [], true)
 5 APPPATH\Controllers\GestionController.php(74): view('ModifierUtilisateur')
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\GestionController->afficherModifier()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\GestionController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
